components/inline/readme.md

```diff
@@ -0,0 +1,9 @@
+# vwc-inline
+
+## Properties
+
+| Property        | Attribute        | Type                              | Default   | Description                                                                      |
+| --------------- | ---------------- | --------------------------------- | --------- | -------------------------------------------------------------------------------- |
+| `columnSpacing` | `column-spacing` | `"sm" \| "md"`                    | "md"      | Sets the initial preferred spacing of a column from predefined available options |
+| `columnBasis`   | `column-basis`   | `"sm" \| "md" \| "lg" \| "block"` | "sm"      | Sets the initial preferred measure of a column from predefined available options |
+| `template`      | `template`       | `"fill" \| "fit"`                 | undefined | Sets the initial preferred template from predefined available options            |
```

Comment:
what do you think of `Auto-Sizing`? is it a better name? template is too generic in the grid sense